package.json

```diff
@@ -41,6 +41,7 @@
     "gulp-sourcemaps": "^1.6.0",
     "gulp-uglify": "^1.4.2",
     "handlebars": "^4.0.3",
+    "localforage": "^1.4.0",
```

Comment:
localStorage is not allowed in service worker
indexedDB... I can write a promisify for it, but would it be simpler?.